client/app/scripts/superdesk-content-filters/content-filters.js

```diff
@@ -69,7 +69,11 @@
         this.getGlobalContentFilters = function() {
             return api.query('content_filters', {'is_global': true})
                 .then(angular.bind(this, function(params) {
```

Comment:
`angular.bind()` can be removed here, the bound function does not make any use of `this` in its body.